lib/Module/Starter/Smart.pm

```diff
@@ -170,11 +170,12 @@ sub create_t {
 }
 
 sub _create_t {
-    my $self = shift;
+    my $self     = shift;
+    my $testdir  = @_ == 2 ? 't' : shift;
```

Comment:
Hello Ruey-Cheng, thanks for your quick review. I am not sure I completely understand what you mean. The second argument will either be the name of the directory the test script should be created in or the filename of the test script, depending on the version of Module::Starter::Simple. For reference, I have added both the old and new protocol of `_create_t` to the PR description (and I should have done so in the first place). This will probably clear it up, but in case it does not please let me know.